cmd/kyma/alpha/create/module/module.go

```diff
@@ -39,6 +44,7 @@ This command creates a component descriptor in the descriptor path (./mod as a d
 Optionally, you can create additional layers with contents in other paths.
 
 Finally, if a registry is provided, the created module is pushed.
```

Comment:
@NHingerl the point is, we have a validation that is always performed when the module is pushed - it's mandatory. For now, it's the "default CR" validation. Maybe it will be extended in the future, but for now it's that one.

Reply:
```suggestion
Finally, if you provided a registry to which to push the artifact, the created module is pushed and validated. For example, the default CR defined in the \"default.yaml\" file is validated against CustomResourceDefinition.
```